tools/hashicorp/Makefile

```diff
@@ -34,13 +34,13 @@ product: ${ARTIFACTS} ## Prepares KM packaging artifacts (e.g. km tarball and kk
 	echo $(MAKE) MAKEFLAGS=$(MAKEFLAGS) -C ${TOP} release
 
 vm-images: .check_packer .check_tools product ## loads source boxes and creates boxes with KM installed
-	@set -x ; for os in ${SRC_OS} ; \
+	@set -ex ; for os in ${SRC_OS} ; \
```

Comment:
is this intentional.

Reply:
propagates errors to the make - otherwise the errors are silent. That's why we didn't notice bix build failure